packages/edit-navigation/src/components/inspector-additions/index.js

```diff
@@ -49,6 +49,12 @@ export default function InspectorAdditions( {
 					openModal={ openManageLocationsModal }
 				/>
 			</PanelBody>
+			<PanelBody>
+				<DeleteMenuPanel
```

Comment:
I see this button has moved to match the design in the issue.

I think the design below is ever so slightly more recent though, which shows it in the Menu settings 😄 :
https://github.com/WordPress/gutenberg/issues/28864#issuecomment-780083954